tinygrad/codegen/linearizer.py

```diff
@@ -259,11 +259,6 @@ def calc_tc_idxs(local_size: int, aliases: List[List[int]]):
                 full_var_sz *= next_var.max+1
             replace_idxs.append(full_var)
           return replace_idxs
-        replace_acc_idxs = calc_tc_idxs(tc.thread_local_sizes[2], tc.thread_local_aliases[2])
```

Comment:
Why is this removed?

Reply:
before, afaict the only thing that this did was permute the output axes for hip tc to work around upcasts always coming before locals (it was building a 16-length index on a 8-length shape dimension, even). now we specify the output pattern through optops.

maybe i missed something about why this was here before.